trade-finance-manager-api/src/v1/api.js

```diff
@@ -214,7 +214,8 @@ const updateDeal = async (dealId, dealUpdate) => {
 
     return response.data;
   } catch (error) {
-    return error;
```

Comment:
There seem to be quite a few other method in this file that have
```
catch(error) {
  return error;
}
```
But that haven't been flagged by the code scanning alerts.

Should I go through and change them to follow this pattern anyway? Or are we safe to assume that the code scanning should have picked up any that we need to worry about?

Reply:
Can you check a couple of these and just make sure that it isn't then returned by the controller that has called this api.js file please? And if it's not then we should be happy to assume the code scanning is doing it's job properly :)

Reply:
If you can go through them as well please that will be great, ideally if they can have the same `console.error` and return statement that will be great.